lib/bamboo/adapters/test_adapter.ex

```diff
@@ -0,0 +1,55 @@
+defmodule BambooSMTP.TestAdapter do
```

Comment:
Should we add a mention about this module somewhere in the readme? 🤔

Reply:
I just made a small update on it as an optional step on the setup.